### ISSUE TITLE
escape { sign

### ISSUE THREAD

**Issue by sakirayan78** [dmester/sffjs#1]
can u tell me how can escape  { sign in text.thankx

**Comment by dmester** [comment]
Curly braces are escaped by doubling the character. E.g.:

``` javascript
String.format("if ({0}) {{ return true; }}", "myVar");
```

will output:

```
if (myVar) { return true; }
```

**Comment by sakirayan78** [comment]
thank you response.but seems not working on my code.here is the my code   var html = "<style>.{classname}  {{position:fixed;top:{height}px;left: 50%; margin-left: -150px;width: 300px;z-index: 9999;}}</style>" ;

   return String.format(html, {type:type,message:message,classname:this.className,height:"87"});

**height property not printed.
thank u

**Comment by dmester** [comment]
I see what you mean. It seems like `{position` is treated as a placeholder even if it is escaped. I will have a look at it and get back to you. Thanks for reporting this issue!

**Comment by dmester** [comment]
This issue should now be fixed in version 1.10. I have tested the format string you used and it seem to work in the new version. Please get back if you still having problem.

**Comment by sakirayan78** [comment]
I confirm.I just downloaded new version,It seems it works.Thank u for quick response and fixing.
